ts/store/reducers/entities/readTransactions.ts

```diff
@@ -0,0 +1,39 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { values } from "lodash";
+import { getType } from "typesafe-actions";
+import { Action } from "../../actions/types";
+import { readTransaction } from "../../actions/wallet/transactions";
+import { GlobalState } from "../types";
+
+// Search transaction number in Set of read transactions
+export const isReadTransaction = (state: GlobalState, txid: number) =>
+  state.entities.transactionsRead[txid] !== undefined;
+
+// filter only unread transactions to account for the residual number
+export const getUnreadTransactions = (state: GlobalState) =>
```

Comment:
Please add memoization or this will get calculated every time the selector is used!

Reply:
How would you memoize the array of unreadTransactions with the current setup?
A way could be to save the return un-potted from getUnreadTransactions, save it in some arbitrary long-term state and proceeding with update on "readTransaction" action but in that case I would have to modify the state definition as well because in the reducer actually used in readTransactions, the part of state passed doesn't include the state that should be updated (unless I create a new one in the same section) 
If this is what you thought i can proceed in this manner but I'm not totally sure it would result in a performance increase.

Reply:
Fixed point following suggestions with the last commit